test/scsi_defs_test.cc

```diff
@@ -32,4 +35,12 @@ namespace {
     EXPECT_EQ(0b11, cb.vendorSpecific);
   }
 
+  /*
+    Tests the TestUnitReadyCommand sruct
+  */
+  TEST(TestUnitReadyCommandStruct, ShouldBePacked) {
```

Comment:
We can replace this test with a static assert (see https://github.com/googleinterns/scsi2nvme/pull/24#discussion_r427580382)